theano/tensor/nnet/nnet.py

```diff
@@ -2335,3 +2335,30 @@ class of the input x while the second predicts the output of the input x in
         output_probs = target_class_probs * output_probs
 
     return output_probs
+
+
+def elu(x, alpha=1):
+    """
+    Compute the element-wise exponential linear activation function.
+
+    .. versionadded:: 0.8.0
+
+    Parameters
+    ----------
+    x : symbolic tensor
+        Tensor to compute the activation function for.
+    alpha : scalar
+
+
+    Returns
+    -------
+    symbolic tensor
+        Element-wise exponential linear activation function applied to `x`.
+
+    References
+    -----
+    .. [1] Djork-Arne Clevert,  Thomas Unterthiner, Sepp Hochreiter
+        "Fast and Accurate Deep Network Learning by
+        Exponential Linear Units (ELUs)" <http://arxiv.org/abs/1511.07289>`.
+    """
+    return alpha * (tensor.exp(0.5 * (x - abs(x))) - 1) + 0.5 * (x + abs(x))
```

Comment:
The formula here seem more complicated then the one used in lasagne:
https://github.com/Lasagne/Lasagne/blob/master/lasagne/nonlinearities.py#L238

Is there a reason? Do you have an idea why they didn't included alpha? Can you confirm they are equivalent when alpha is 1 and if so, make an in here and use there simpler code for that special case? I think it will generate better code.